### PR TITLE
Add limitations to documentation

### PR DESCRIPTION
**Pull request by nunocosta-casafari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,25 @@
 Flake8 plugin to check explicitly passed arguments.
 
+<!-- toc -->
```
Comment (iurycasafari) [inline README.md]: this is necessary?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+  * [Tests](#tests)
+- [Considerations](#considerations)
+  * [Positional only parameters](#positional-only-parameters)
+    + [Built-in functions](#built-in-functions)
+    + [Extension modules](#extension-modules)
+    + [Parameters defined as positional only](#parameters-defined-as-positional-only)
+- [Limitations](#limitations)
+- [References](#references)
+
+<!-- tocstop -->
```
Comment (iurycasafari) [inline README.md]: this is necessary?

File: README.md (hunk 1 above)
Comment (nunocosta-casafari) [reply README.md]: No. 

But if we use this https://github.com/jonschlinkert/markdown-toc it makes our life much easier :stuck_out_tongue:

Comment (anagilda) [reply README.md]: Is this also what has changed the bullets to `-` and `+`?

Comment (nunocosta-casafari) [reply README.md]: Yes. But there is an option to select the _bullets_. I fix that now with it.

File: README.md [hunk 3]
```diff
@@ -83,3 +94,43 @@ Now you can run the tests by with `tox`.
...
+
+```python
+def is_ancestor(node_1, node_2, /, path_length):
+    ...
+```
+
+# Limitations
+
+* Implicit arguments for _positional-only parameters_ -- whose arguments can't be explicit -- are marked as error, unless it's a built-in function.
+* Implicit arguments for _positional-or-keyword parameters_ -- whose arguments can be explicit -- in built-in functions
```
Comment (anagilda) [inline README.md]: Use only one dash `-` instead of two.

**Pull request merged**